apps/src/netsim/NetSimBandwidthControl.js

```diff
@@ -8,172 +8,40 @@
  maxlen: 90,
  maxstatements: 200
  */
-/* global $ */
 'use strict';
 
-var markup = require('./NetSimBandwidthControl.html');
-var i18n = require('../../locale/current/netsim');
-
-/**
- * Min value of 2 is 2^2 or 4bps
- * @type {number}
- * @const
- */
-var SLIDER_MIN_VALUE = 2;
-
-/**
- * Slider value is used as the exponent,
- * so max value of 18 here is actually 2^18 or 256Kbps
- * Since the top value is treated as Infinity, that means the max values are:
- *   ---64Kbps----128Kbps----Infinity-|
- * @type {number}
- * @const
- */
-var SLIDER_MAX_VALUE = 18;
-
-/**
- * @type {number}
- * @const
- */
-var BITS_PER_KILOBIT = 1024;
-
-/**
- * @type {number}
- * @const
- */
-var BITS_PER_MEGABIT = 1024 * BITS_PER_KILOBIT;
-
-/**
- * @type {number}
- * @const
- */
-var BITS_PER_GIGABIT = 1024 * BITS_PER_MEGABIT;
+require('../utils');
```

Comment:
With a module that we're requiring but not assigning to anything, it might be worth a comment about why we need it? (my guess is for inherits?)